api_validation/src/main/scala/com/nvidia/spark/rapids/api/ApiValidation.scala

```diff
@@ -69,6 +69,16 @@ object ApiValidation extends Logging {
     val gpuKeys = gpuExecs.keys
     var printNewline = false
 
+    val sparkToShimMap = Map("3.0.0"->"spark300", "3.0.1"->"spark301", "3.1.0"->"spark310")
+    val sparkVersion = ShimLoader.getSparkShims.getSparkShimVersion.toString
+    var shimVersion = sparkToShimMap(sparkVersion)
+    // There is no separate implementation for Execs in spark-3.0.1.
+    shimVersion = if (shimVersion == "spark301") {
```

Comment:
seems fine for now, if this gets to complex we may just make shim functions to get the class but we can wait and do that later if needed

Reply:
Sounds good.